README.md

```diff
@@ -60,6 +60,24 @@ visit your app at `http://localhost:4200/?fastboot=false`. If you want to turn o
 
 You can even disable serving fastboot with `ember serve` using an environment flag: `FASTBOOT_DISABLED=true ember serve`. If you have disabled building fastboot assets using the same flag as described [here](https://github.com/ember-fastboot/ember-cli-fastboot#double-build-times-and-no-incremental-builds), remember to also disable serving fastboot assets when using `ember serve`.
 
+### FastBoot Configuration
+
+When running locally using `ember serve` you can pass options into FastBoot instance via `config/fastboot.js` file.
+
+```js
+module.exports = function(environment) {
+  let myGlobal = environment === 'production' ? process.env.MY_GLOBAL : 'testing';
+
+  return {
+    sandboxGlobals: {
+      myGlobal;
+    }
+  };
+}
+```
+
```

Comment:
The current code path will not merge the `myGlobal` in production I believe. So do you need to define production? It is primarily for `ember s` only.

Reply:
Yeah, this might be little bit confusing - the environment is provided by `ember-cli`, hence the `ember s -prod` will also set environment to `production`.

Where this file will not work is when used with `fastboot-app-server`, but this is different setup - should be the README updated with mention about `fastboot-app-server` (the example usage can also be provided)?

Reply:
Yes please it may be good to specific it as it will help consumers not assume this will work when using `fastboot-app-server`.

Reply:
👍, I've added section about FastBoot App Server below, you can take a look.